Test failure with mosquitto extension

Without, test suite passes

With mosquitto 0.4.0 enabled:
```
PHPUnit 6.2.3 by Sebastian Bergmann and contributors.

.....AssertionError: The name cannot be empty in /tmp/phan/src/Phan/Language/FQSEN/FullyQualifiedGlobalStructuralElement.php:117
Stack trace:
#0 /tmp/phan/src/Phan/Language/FQSEN/FullyQualifiedGlobalStructuralElement.php(117): assert(false, 'The name cannot...')
#1 /tmp/phan/src/Phan/Memoize.php(76): Phan\Language\FQSEN\FullyQualifiedGlobalStructuralElement::Phan\Language\FQSEN\{closure}()
#2 /tmp/phan/src/Phan/Language/FQSEN/FullyQualifiedGlobalStructuralElement.php(133): Phan\Language\FQSEN\FullyQualifiedGlobalStructuralElement::memoizeStatic('Phan\\Language\\F...', Object(Closure))
#3 /tmp/phan/src/Phan/Language/Element/GlobalConstant.php(60): Phan\Language\FQSEN\FullyQualifiedGlobalStructuralElement::fromFullyQualifiedString('\\0')
#4 /tmp/phan/src/Phan/CodeBase.php(270): Phan\Language\Element\GlobalConstant::fromGlobalConstantName(Object(Phan\CodeBase), '0')
#5 /tmp/phan/src/Phan/CodeBase.php(162): Phan\CodeBase->addGlobalConstantsByNames(Array)
#6 /tmp/phan/tests/Phan/PhanTestListener.php(51): Phan\CodeBase->__construct(Array, Array, Array, Array, Array)
#7 /tmp/phan/vendor/phpunit/phpunit/src/Framework/TestResult.php(395): Phan\Tests\PhanTestListener->startTest(Object(Phan\Tests\PhanTest))
#8 /tmp/phan/vendor/phpunit/phpunit/src/Framework/TestResult.php(637): PHPUnit\Framework\TestResult->startTest(Object(Phan\Tests\PhanTest))
#9 /tmp/phan/vendor/phpunit/phpunit/src/Framework/TestCase.php(883): PHPUnit\Framework\TestResult->run(Object(Phan\Tests\PhanTest))
#10 /tmp/phan/vendor/phpunit/phpunit/src/Framework/TestSuite.php(746): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#11 /tmp/phan/vendor/phpunit/phpunit/src/Framework/TestSuite.php(746): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 /tmp/phan/vendor/phpunit/phpunit/src/Framework/TestSuite.php(746): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#13 /tmp/phan/vendor/phpunit/phpunit/src/Framework/TestSuite.php(746): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#14 /tmp/phan/vendor/phpunit/phpunit/src/TextUI/TestRunner.php(537): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#15 /tmp/phan/vendor/phpunit/phpunit/src/TextUI/Command.php(210): PHPUnit\TextUI\TestRunner->doRun(Object(PHPUnit\Framework\TestSuite), Array, true)
#16 /tmp/phan/vendor/phpunit/phpunit/src/TextUI/Command.php(141): PHPUnit\TextUI\Command->run(Array, true)
#17 /usr/bin/phpunit(52): PHPUnit\TextUI\Command::main()
#18 {main}
```

--- a/src/Phan/CodeBase.php
+++ b/src/Phan/CodeBase.php
@@ -249,6 +249,11 @@ class CodeBase
     private function addGlobalConstantsByNames(array $const_name_list)
     {
         foreach ($const_name_list as $const_name) {
+            if (!$const_name) {
+                // #1015 workaround for empty constant names ('' and '0').
+                fprintf(STDERR, "Saw constant with empty name of %s. There may be a bug in a PECL extension you are using (php -m will list those)\n", var_export($const_name, true));
+                continue;
+            }
             try {
                 $const_obj = GlobalConstant::fromGlobalConstantName($this, $const_name);
                 $this->addGlobalConstant($const_obj);

